lib/pages/BottomNav.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'Home.dart';
+import 'Cases.dart';
+import 'Stat.dart';
+import 'Symptoms.dart';
+import 'Contact.dart';
+
+class BottomNav extends StatefulWidget {
+  @override
+  _BottomNavState createState() => _BottomNavState();
+}
+
+class _BottomNavState extends State<BottomNav> {
```

Comment:
Change `BottomNav` to `BottomNavigation`

Reply:
Goods na paps.